docs/.vuepress/config.js

```diff
@@ -570,6 +589,15 @@ module.exports = {
 		},
 	},
 	plugins: [
+		[
+			require('./typedoc/plugin.js'),
+			{
+				entryPoints: ['../api/src/index.ts'],
+				tsconfig: '../api/tsconfig.json',
+				out: 'generated',
+				publicPath: '/generated/',
+			},
+		],
```

Comment:
I closed an issue regarding dynamically generated docs for the Vue app... then re-opened it because if I read through this correctly typedoc is just for TS stuff and the entry point is `api` and doesn't touch the `app` directory.

Am I understanding that correctly?
No Vue component documentation?
Will we want another solution for the Vue wiki?

Reply:
Thats correct this just generates TS docs more specifically the services that are exposed to extensions and does not touch the `app` directory for generated vue component docs we will need another solution.